backend/spec/models/group_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Group do
+  subject(:group1) { described_class.create(group_params) }
+
+  let(:current_user) { create(:user) }
+  let(:name) { 'SAWQ' }
+  let(:user_id) { current_user.id }
+  let(:group_params) { { name:, user_id: } }
+
+  it 'creates a group' do
+    expect(group1).to be_persisted
+  end
+
+  it 'realy creates a group' do
```

Comment:
The name of this test is not clear enough. You could go with something like "increments amount of records in the database by one"